actors/miner/src/monies.rs

```diff
@@ -40,6 +40,16 @@ pub const TERMINATION_REWARD_FACTOR_DENOM: u32 = 2;
 const LOCKED_REWARD_FACTOR_NUM: u32 = 3;
 const LOCKED_REWARD_FACTOR_DENOM: u32 = 4;
 
+pub const TERM_PENALTY_PLEDGE_NUM: u32 = 85;
+pub const TERM_PENALTY_PLEDGE_DENOM: u32 = 1000;
+
+// TODO: what is a good value? should it be per network?
+pub const MIN_TERMINATION_FEE_PLEDGE_NUM: u32 = 2;
+pub const MIN_TERMINATION_FEE_PLEDGE_DENOM: u32 = 100;
+// TODO: what is a good value? should it be per network?
+pub const FAULT_FEE_MULTIPLE_NUM: u32 = 5;
```

Comment:
It should be 105. In the current implementation, it’s actually max{% of IP, 0.05*FF}, which seems to be a bug.
```suggestion
pub const FAULT_FEE_MULTIPLE_NUM: u32 = 105;
```

Reply:
I believe you are right; I took 5% from a conversation with @Schwartz10, but I misunderstood and he meant 105%. This suggestion to the FIP also confirms it: https://github.com/filecoin-project/FIPs/pull/1128#discussion_r1966944582.